client/assets/icons/down-arrow-stem.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
+  "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<!-- Created with Inkpad (http://www.taptrix.com/) --><svg xmlns:inkpad="http://taptrix.com/inkpad/svg_extensions" height="159pt" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg" width="159pt" version="1.1" viewBox="0,0,159,159">
+<defs/>
+<g id="Untitled" inkpad:layerName="Untitled">
+<path d="M65.2232-0.523205L65.2232+99.2064L22.628+56.6104L0+79.2392L79.7616+159L159.523+79.2392L136.895+56.6104L94.3008+99.2064L94.3008-0.523205L65.2232-0.523205Z" opacity="1" fill="#000000"/>
```

Comment:
Thanks for using `.svg` :)

Reply:
Haha! Yeah. I hated using PNGs for the stars, but the quarter-star issue forced my hand as there are no unicodes for those. SVG can work for that, but it is involved enough (found an article) I figured I'd skip it for MVP and hopefully circle back later to do it better :-D